python/trezorlib/client.py

```diff
@@ -112,6 +112,7 @@ def __init__(self, transport, ui=_NO_UI_OBJECT, state=None):
         self.transport = transport
         self.ui = ui
         self.state = state
+        self.check_firmware = True
```

Comment:
let's not make it _this_ easy for 3rd parties to skip the version check.

we might do what Electrum does (`TrezorClient.is_outdated = lambda self: True`)

or better, we can override `trezorlib.MINIMUM_FIRMWARE_VERSION` value in the test runner

Reply:
Yes, I agree. I've added this option to set this to False in debug client only, but I understand 3rd parties could still do so.

> or better, we can override trezorlib.MINIMUM_FIRMWARE_VERSION value in the test runner

I like that. Will do